go/vt/vtctl/vtctl.go

```diff
@@ -478,7 +478,7 @@ var commands = []commandGroup{
 		"Workflow", []command{
 			{"Workflow", commandWorkflow,
 				"<ks.workflow> <action> --dry-run",
-				"Start/Stop/Delete/List/ListAll Workflow on all target tablets in workflow. Example: Workflow merchant.morders Start",
+				"Start/Stop/Delete/Show/ListAll Workflow on all target tablets in workflow. Example: Workflow merchant.morders Start",
```

Comment:
While we're renaming things, does ListAll really list all, or only some (e.g. only active workflows)? Is there any reason we can't reuse the name List?

Reply:
@sougou felt that this command should replace `ShowResharding` and suggested giving it a similar idiomatic name of `Show`.

Reply:
FTR, I don't feel strongly about one way or another. if there are stronger arguments to go with `List`, we can go with that.

Reply:
List -> Show sounds good. I think @enisoc is suggesting ListAll -> List, and I agree.

Reply:
However, ListAll -> List would be out of scope of this PR, no?

Reply:
That can be a separate PR.

Reply:
Yes I meant ListAll->List. I agree that can be a separate PR.